.github/actions/check-app/action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Check application'
+description: 'Check if application is deployed on AKS cluster'
+inputs:
+  azure-credentials:
+    description: 'Azure credentials to sign in'
+    required: true
```

Comment:
I suggest the name `verify-app-availability`.

Reply:
Good point. Fixed in 99e770b.